_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SQL Workshop"
+description: "SQL Workshop Intro"
+date: 2019-07-25T13:24:17-07:00
+prereq: "None"
+difficulty: "Beginner"
+image: ""
+draft: false
+# icon: "fab fa-python"
+weight: 130
+# translationKey: "SQL-workshop-home"
+---
+
+# Introduction: Learn SQL by Saving the Galaxy!
+
+
+![Galaxy](media/Galaxy.png)
+
+<font size="5">Welcome to the Nuevo Space Station (NSS)! 
+
+<font size="3">You are a space adventurer who travels
+ the galaxy in your speedy rocket ship while solving coding puzzles to help save your
+ alien friends on different planets.
+
+You first need to apply as an honorary space explorer so you can start your
+adventures:
+
+[Space Cadet Survey](https://docs.google.com/forms/d/e/1FAIpQLSfTEKYwDtkwfJFfYOclEHJQSX2R9xIaPScSorowQptY3lREzw/viewform)
```

Comment:
what is this for? what do we do with the results of the form?

Reply:
We are planning on making a database from the results of the form. It asks students questions like "what is your favorite color" and at the (optional) ending, we ask students to search through the survey database to find out how many students like the same color as them.